components/support/ktx/src/main/java/mozilla/components/support/ktx/kotlin/String.kt

```diff
@@ -290,3 +295,10 @@ fun String.getRepresentativeCharacter(): String {
 fun String.last4Digits(): String {
     return this.takeLast(LAST_VISIBLE_DIGITS_COUNT)
 }
+
+/**
+ * Returns truncated urls and search terms to improve performance.
```

Comment:
```suggestion
 * Returns a trimmed string. This is used to prevent extreme cases
 * from slowing down UI with large strings.
```
Maybe we should name this function `String.trimmed()` since it isn't just about URLs